### PR TITLE
Add sort to action user inputs

### PR DESCRIPTION
**Pull request by Eyal-Be**
# Description

What - added sort to self service actions user inputs
Why - to enable entities selection sort in the UI
How - added sort to relevant models

## Type of change

Please leave one option from the following and delete the rest:

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: port/action/resource_test.go [hunk 1]
```diff
@@ -2192,3 +2192,73 @@ func TestRequiredApprovalAll(t *testing.T) {
 		},
 	})
 }
+
```
Comment (erikzaadi) [inline port/action/resource_test.go:2195]: In one of the other tests, please add an assertion that the sort property isn't there

File: examples/resources/port_action/main.tf [hunk 2]
```diff
@@ -88,6 +88,16 @@ resource "port_action" "restart_microservice" {
           default     = "https://example.com"
           pattern     = "^https://.*"
         }
+        service = {
```
Comment (erikzaadi) [inline examples/resources/port_action/main.tf:91]: Could you run `terraform fmt` on this pretty plz?

File: port/action/schema.go [hunk 3]
```diff
@@ -1111,6 +1153,10 @@ resource "port_action" "create_microservice" {
                             }]
                         })
 					}
+                    sort = {
```
Comment (erikzaadi) [inline port/action/schema.go:1156]: Is it me or is the indent here a bit askew?

**Review by erikzaadi — CHANGES_REQUESTED** [review]
2.5 small NITs

File: port/action/resource_test.go (hunk 1 above)
Comment (Eyal-Be) [reply port/action/resource_test.go:2195]: done

File: examples/resources/port_action/main.tf (hunk 2 above)
Comment (Eyal-Be) [reply examples/resources/port_action/main.tf:91]: check how it looks locally

File: port/action/schema.go (hunk 3 above)
Comment (Eyal-Be) [reply port/action/schema.go:1156]: check how it looks locally

**Review by erikzaadi — APPROVED** [review]

**Pull request merged**